### PR TITLE
PUB-1419 PartyRole mappings

### PR DESCRIPTION
**Pull request by joshblackmoor**
### JIRA link ###

https://tools.hmcts.net/jira/browse/PUB-1419

### Change description ###

Added mapping for party roles.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/service/publicationService.ts [hunk 1]
```diff
@@ -252,4 +254,16 @@ export class PublicationService {
     }
     return publishedTime;
   }
+
+  private static convertPartyRole(nonConvertedPartyRole: string): string {
```
Comment (ChrisS1512) [inline src/main/service/publicationService.ts:258]: Need some unit tests for this bit :)

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/service/publicationService.ts (hunk 1 above)
Comment (joshblackmoor) [reply src/main/service/publicationService.ts:258]: 👍 Have added a test to cover a list that has values to be mapped (sonar is now showing 100% coverage for this PR). 
I've also refactored the function again to only use one if as realised it was unnecessary for there to be two separate ones.

Comment (ChrisS1512) [reply src/main/service/publicationService.ts:258]: Really nice :)

**Review by ChrisS1512 — APPROVED** [review]

**Review by furnivall — APPROVED** [review]
Great code, I love it. Such sexy code.

**Review by ChrisS1512 — APPROVED** [review]

**Review by furnivall — APPROVED** [review]

**Pull request merged**